FastSimulation/Configuration/python/Reconstruction_AftMix_cff.py

```diff
@@ -51,14 +51,62 @@
 # Changes to tracking sequences
 ##########################################
 # modules to be removed
-_mod2del = _reco.trackingGlobalReco.expandAndClone()._seq._collection
+_mods = []
+_visitor = ModuleNodeVisitor(_mods)
+_reco.recopixelvertexing.visit(_visitor)
+_reco.trackingGlobalReco.visit(_visitor)
+
+# actually we want to keep a few modules that we need to run (again) after mixing) 
+_entry = [_reco.firstStepPrimaryVerticesUnsorted,_reco.firstStepPrimaryVertices,_reco.ak4CaloJetsForTrk,_reco.caloTowerForTrk,_reco.initialStepTrackRefsForJets,_reco.trackExtrapolator]
+
+for _key,_value in _reco.__dict__.items():
+    if _value in _mods and _value not in _entry:
+        delattr(_reco,_key)
```

Comment:
This is removing the items from the Configuration.StandardSequences.Reconstruction_cff python module, which is what we do not want to do.

Reply:
That is what is done in the original version.